keps/693-multikueue/README.md

```diff
@@ -188,16 +194,28 @@ type MultiKueueClusterStatus {
    Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
 }
 ```
+### Subcomponents
 
-MultiKueue controller will monitor all cluster definitions and maintain 
+#### MultiKueueCluster Controller
+
+Will monitor all cluster definitions and maintain 
 the Kube clients for all of them. Any connectivity problems will be reported both in
-MultiKueueCluster status as well as AdmissionCheckStatus and Events. MultiKueue controller 
+MultiKueueCluster status and Events. MultiKueue controller 
 will make sure that whenever the kubeconfig is refreshed, the appropriate 
-clients will also be recreated.
+clients will also be recreated and attempt to reconnect when the connection to the
+target cluster is lost.
 
 Creation of kubeconfig files is outside of the MultiKueue scope, and is cloud
 provider/environment dependant.
 
+#### AdmissionCheck Controller
+
+Will monitor the AdmissionChecks associated with MultiKueue (by ControllerName) and maintain
+their `Active` status condition based on the validity of their MultiKueueConfig and `Active`
+state of the MultiKueueClusters in use.
+
+#### Workload Controller
```

Comment:
Add a sentence or two to summarize the overall responsibility of this subcomponent.

Reply:
done